apps/examples/src/examples/hosted-images/README.md

```diff
@@ -6,3 +6,10 @@ priority: 2
 hide: true
```

Comment:
not sure why this one is hidden but maybe remove this?

Reply:
Interestingly, we don't hide ANY examples within tldraw.dev, and that's what I was looking at improving. But either way, yes, will remove this